src/cmp_aus.js

```diff
@@ -95,31 +95,90 @@ const reloadPage = async (page) => {
 };
 
 const checkPrebid = async (page) => {
-	log(`Reloading page: Start`);
+	// --------------- RELOAD PAGE START ---------------------------
+	log(`[TEST 4: RELOAD PAGE] Step start`);
 	const reloadResponse = await page.reload({
 		waitUntil: 'domcontentloaded',
 		timeout: 30000,
 	});
 	if (!reloadResponse) {
-		logError(`Reloading page : Failed`);
+		logError(`[TEST 4: RELOAD PAGE] Reloading page : Failed`);
 		throw 'Failed to refresh page!';
 	}
-	log(`Reloading page: Complete`);
+	log(`[TEST 4: RELOAD PAGE] Step complete`);
+	// --------------- RELOAD PAGE END ---------------------------
+
+	// --------------- BUNDLE START ---------------------------
+	log(`[TEST 4: PREBID BUNDLE] Checking: graun.Prebid.js.commercial.js`);
+	await page.waitForRequest((req) =>
+		req.url().includes('graun.Prebid.js.commercial.js'),
+	);
+	log(`[TEST 4: PREBID BUNDLE] Step start`);
+	// --------------- BUNDLE END ---------------------------
+
+	// --------------- CANADA START ---------------------------
+	log(`[TEST 4: CANADA] Step start`);
```

Comment:
We don't run checks in the canada region do we?

Reply:
We do afaik

Reply:
Ah my mistake!